index.html

```diff
@@ -4,272 +4,289 @@
     <title>Portfolio</title>
   </head>
   <body>
-    <main>
-      <header class="top-nav-burger">
-        <div class="top-corner-image">
-          <img
-            src="/resources/images/image_geometry_1.png"
-            alt="corner image"
-          />
+    <header class="nav-menu-top">
+      <nav class="desktop-nav-top">
+        <ul class="desktop-nav-list">
+          <li>Hello</li>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+      <nav class="menu-top">
+        <div class="background-image-nav"></div>
+        <div class="burger-image-mobile">
+          <a href="#">
+            <img
+              class="burguer-image"
+              src="/resources/images/ic_menu.png"
+              alt=""
+            />
+          </a>
         </div>
-        <nav class="container burger">
-          <div class="menu-burger-image">
-            <img src="/resources/images/ic_menu.png" alt="burguer menu" />
+      </nav>
+    </header>
+    <main>
+      <section class="about-me">
+        <div class="about-me-mobile">
+          <div class="about-me-header">
+            <h1>
+              Charlie <br />
+              Giorgania
+            </h1>
           </div>
-        </nav>
-      </header>
-      <section class="container introduction">
-        <div>
-          <h1 class="developers-name">Charlie Giorgania</h1>
-          <p class="intro-content">
-            Hello! I am a software developer! I can help you build a product,
-            feature or website. Take a look of my works. If you like what you
-            see and have a project you need coded, don’t hesitate and contact
-            me.
-          </p>
-        </div>
-        <div class="colab-detail">
-          <div class="colab-butn">
-            <a href="#" class="start-colab colab-butn colab-detail"
+          <div class="about-me-paragraph">
+            <p>
+              Hello! I am a software developer! I can help you build a product,
+              feature or website. Taka a look of my works. If you like what you
+              see and have a project you need coded, don’t hesitate and contact
+              me.
+            </p>
+          </div>
+          <div class="button-top">
+            <a class="get-my-cv-btn main-button button-action" href="#"
               >Start collaboration</a
             >
           </div>
         </div>
-      </section>
-    </main>
-    <section class="projects-main">
-      <div class="projects-header">
-        <h2 class="main-topic topic-h2">Projects</h2>
-      </div>
-      <div class="grd-container works-catalog">
-        <div class="project-intro">
+        <div class="about-me-background">
           <img
-            src="/resources/images/computer-hands.png"
-            alt="hands on computer"
-            class="section-image"
+            src="/resources/images/mainbackground.png"
```

Comment:
[ Optional Changes ]

- [ ] Kindly work on all images path in your project. All your images seem to not rendering and are broken. You can you the format below to trace the path of your images `src="./resources/images/image_geometry_1.png"`
- [ ] Please check all your images and fix that issue. Also, please don't forget the CSS, you used images there as well. 👍